pkg/events/events.go

```diff
@@ -6260,6 +6260,11 @@ var Definitions = eventDefinitions{
 					{EventID: DoInitModule},
 				},
 				KSymbols: &[]kSymbolDependency{},
+				Capabilities: capDependency{
+					capabilities.Base: []cap.Value{
+						cap.SYSLOG, // read /proc/kallsyms
```

Comment:
Where do you need this capability?
Why can't we simply raise it where required and not add it to the base ring?

Reply:
We've done it like this so far for the other uprobe events. I don't disagree with you in principle but we should probably fix it across the board rather than just here in particular.